cds_ils/config.py

```diff
@@ -457,7 +457,7 @@ def _parse_env_bool(var_name, default=None):
             "marshmallow": SanitizedUnicode(required=True),
         },
         "standard_review:checkdate": {
-            "elasticsearch": "date",
```

Comment:
I didn't check all records but some of them doesn't have a date in the field 925__z. For example, this [record](https://cds.cern.ch/record/2690790/export/xm?ln=en) it's value is `Reviewed in December 2019`

Reply:
then it should raise an error when we import, because the data is a wrong type. We shouldn't change our new data model because of the inconsistency in the old system